data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,530 @@
+			label true
+			`	"Will you take me to Hai-home?" he asks. "You are able to be outside Unfettered places. Can I go with you?"`
+			choice
+				`	"Yes, I will take you there."`
+					goto end
+				`	"Sorry, I can't risk angering the Unfettered by carrying a defector."`
+			label refuse
+			`	Disappointed, the youth leaves. You hope you made the right choice.`
+				decline
+			label end
+			`	You show the youth to one of your bunk rooms, and tell him to stay hidden there until you reach Hai-home.`
+				accept
+	
+	on visit
+		dialog `You look for the young Hai, but realize that he took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
+	on complete
+		payment 100000
+		dialog
+			`You need not have worried about the reception the Unfettered youth would receive here on Hai-home. Scarcely minutes after you hesitantly contact the Hai government, a happy crowd has gathered around your ship. When he steps out of the hatchway, he looks stunned at being welcomed so warmly. One of the Hai governors thanks you for transporting him, and pays you <payment>.`
+		"reputation: Hai" += 10
+		"reputation: Hai (Wormhole Access)" += 10
+		"reputation: Hai Merchant" += 10
+		"reputation: Hai Merchant (Human)" += 10
+		"reputation: Hai Merchant (Sympathizers)" += 10
+
+
+
+mission "Hai Legacy 1"
+	invisible
+	shipyard
+	source "Hai-home"
+	destination "Firelode"
+	clearance
+	to offer
+		has "First Contact: Unfettered: offered"
+		# note: this mission chain could be tweaked later to trigger even after they are friendly
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		# it will offer but defer itself the first time if you lack money but have sufficient net worth
+		"credits" + "net worth" * ( 1 - "Hai Legacy 1: deferred" ) >= 10000000
+	on offer
+		conversation
+			branch first
+				not "Hai Legacy 1: deferred"
+			`Do you want to head to the coordinates as the hooded figure suggested?`
+			choice
+				`	(Sure.)`
+					goto go
+				`	(Maybe another time.)`
+					defer
+				`	(Never.)`
+					decline
+			label first
+			`As you're browsing the shipyard, you are discreetly approached by a hooded figure. When you stop to see what they want to say, they grab and pull your arm to follow them. "Keep walking," the stranger hisses. After pointing at and commenting on random ship, as if to mimic the actions of regular passerby, they address you again. "My associates and I have a lucrative business proposition for you. If you are interested, come find us at these coordinates." Without waiting for a response, they fold a piece of paper into your hand before quickly fading into the crowd. It lists only a location: a beach on the other side of the planet.`
+			`	However, it also mentions you need to have at least 10 million in cash! It may be a good idea to save up or take a debt and then come back another day to head to the location.`
+				to display
+					"credits" < 10000000
+				defer
```

Comment:
```
# it will offer but defer itself the first time if you lack money but have sufficient net worth
		"credits" + "net worth" * ( 1 - "Hai Legacy 1: deferred" ) >= 10000000
```
as said, it'll only happen once if you lack the money bc I set hai legacy 1: deffered in the on defer, so the net worth is ignored after the first time

Reply:
It's not about the money requirements, it's about how it looks like the the dialog leads to the `defer` keyword.